src/models/credit-bundle.ts

```diff
@@ -96,9 +96,9 @@ const getCreditBundleModel = function ({
 			organization: string | number,
 			featureId: number,
 			creditsToPurchase: number,
-		): Promise<PinePostResult<CreditBundle>> => {
+		) => {
 			const orgId = await getOrgId(organization);
-			const body: SubmitBody<CreditBundle> = {
+			const body: CreditBundle['Write'] = {
```

Comment:
optional: feel free to inline such cases, instead of having to explicitly define their type